cmd/snap-mgmt/snap-mgmt.sh.in

```diff
@@ -136,6 +136,9 @@ purge() {
     echo "Removing downloaded snaps"
     rm -rf /var/lib/snapd/snaps/*
 
+    echo "Removing features exported from snapd to helper tools"
+    rm -rf /var/lib/snapd/features
```

Comment:
You probably need to update `postrm` for Ubuntu/Debian packaging too.

Reply:
Actually, we remove all of `/var/lib/snapd` in postrm so this is good as-is.